packages/grap/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://www.lunabase.org/~faber/Vault/software/grap/
+TERMUX_PKG_DESCRIPTION="Grap allow to display graphs in a Groff workflow."
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_MAINTAINER="@xingguangcuican6666"
```

Comment:
You don't need to set yourself as the maintainer for packages you are submitting, but if you want to we prefer this to be done in the Git format of `pseudonym <user@email.ext>`.
The suggestion below uses the email you used for this commit for that.
```suggestion
TERMUX_PKG_MAINTAINER="@xingguangcuican6666 <xingguangcuican666@foxmail.com>"
```